tensorflow_datasets/scripts/generate_visualization.py

```diff
@@ -23,40 +23,82 @@
 
 import os
 from absl import app
+from absl import flags
+from concurrent import futures
 
 import tensorflow as tf
 import tensorflow_datasets as tfds
+from tensorflow_datasets.core import registered
 
+WORKER_COUNT_DATASETS = 200
```

Comment:
200 seems to big as I don't think 200 datasets can be loaded in parallel. Maybe set to 10 ?

Reply:
Okay I will change this